ext/hal/nordic/nrfx_config_nrf52832.h

```diff
@@ -954,28 +954,28 @@
 
 // <e> NRFX_PWM_ENABLED - nrfx_pwm - PWM peripheral driver
 //==========================================================
-#ifndef NRFX_PWM_ENABLED
-#define NRFX_PWM_ENABLED 0
+#ifdef CONFIG_NRFX_PWM
+#define NRFX_PWM_ENABLED 1
```

Comment:
does this need another else clause to make sure that NRFX_PWM_ENABLED is set to 0 if it is not already defined?

eg:
#ifndef CONFIG_NRFX_PWM
#define NRFX_PWM_ENABLED 1
#else 
#ifndef NRFX_PWM_ENABLED
#define NRFX_PWM_ENABLED 0
#endif 
#endif

Reply:
I followed the NRFX_SPIM driver as my example.  I dont think its needed based on how it implements the NRFX_CHECK macro.